### PR TITLE
Update it_IT app translation for 3.8.2  Fixes #2304

### LINKED ISSUE #2304: Update it_IT app translation for 3.8.2
Hi @dzpex,
We are getting ready for the 3.8.2 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **it_IT** by **2022-02-06**.

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-02-06

Suggested commit and PR message:
```
Update it_IT app translation for 3.8.2

Fixes #2304
```

Note: Some of the displayed "untranslated" strings will actually not be new, but will rather be shown due to technical reformatting (e.g. %1). You may be able to easily update them using the "vanished" translations list.

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by dzpex**
**Short description of changes**

**Context: Fixes an issue?**

**Does this change need documentation? What needs to be documented and how?**

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

## Checklist

- [ ] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I tested my code and it does what I want
- [ ] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [ ] I've filled all the content above

### CODE REVIEW

File: src/res/translation/translation_it_IT.ts [hunk 1]
```diff
@@ -583,7 +583,7 @@
     <message>
         <location filename="../../chatdlg.cpp" line="145"/>
         <source>Do you want to open the link &apos;%1&apos; in your browser?</source>
-        <translation>Vuoi aprire il link nel tuo Browser?</translation>
+        <translation>Vuoi aprire il link %1 nel tuo Browser?</translation>
```
Comment (hoffie) [inline src/res/translation/translation_it_IT.ts:586]:
```suggestion
        <translation>Vuoi aprire il link &apos;%1&apos; nel tuo Browser?</translation>
```
?

**Review by hoffie — APPROVED** [review]
PR looks fine to me, thanks! :)
One minor question/suggestion (see details).

- [x] Translator listed in the src/util.cpp, optionally add link to PR or code
- [x] Punctuation and spacing consistent.
- [x] Signal words consistent (“ASIO”, “Buffer”)
- [x] App translations: No untranslated strings (grep unfinished -5 src/res/translation/translation_$TRANSLATION*.ts)
- [x] App translations: Only a single .ts file checked in (.qm in addition is also OK)
- [x] Installer translations: Passes tools/check-wininstaller-translations.sh

Note: Should be squash-merged. Let's wait if @dzpex wants to incorporate the suggestion.

File: src/res/translation/translation_it_IT.ts (hunk 1 above)
Comment (dzpex) [reply src/res/translation/translation_it_IT.ts:586]: for me its ok! you need that i fix the file from my pull request? or is not necessary?

Comment (hoffie) [reply src/res/translation/translation_it_IT.ts:586]: You should be able to click the "Commit suggestion" button directly on Github to create a commit in your PR. :)

**Review by ann0see — APPROVED** [review]

**Pull request merged**